prow/scripts/cluster-integration/create-image.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
This is a summary from our discussion:
I am afraid that this job is doing too many things. 
Additionally, building image is different than for components. 
IMO you should check:
- if it is possible to have a makefile in kyma repository that builds kyma-installer
- if you can have a separate job, only for building kyma-installer. That job will be then similar for other components jobs and additionally, it will specify to run gke-integration job after success (`run_after_success`  param)
IN such approach we are unable to remove kyma-installer image, but IMO this is not a big problem - because for components we also don't remove PR's images.

It is up to @Tomasz-Smelcerz-SAP if he want to do it here, or create follow-up task.

Reply:
Thanks for feedback Adam, we'll look into this ideas. We would have done it already if not for the very broad scope of Kyma-Installer.
It must be built in many scenarios: Pre-releases, Releases, PRs, local deployment, installing additional commercial components and so on.
This means we should take care not to look at Kyma-Installer build only from a perspective of one scenario/job (like this one) because that could occur a dead-end later.

Having said that we definitely don't want to have Kyma-Installer build configurations spread across multiple projects/files, we want to have it in one place, so a follow-up task to introduce such place (probably in Kyma) is necessary. Once we have the concept, we can go back to _this_ job and refactor it correspondingly